docs/api/apiv3/components/examples/hierarchy_item_collection_response.yml

```diff
@@ -0,0 +1,45 @@
+description: |-
+  Simple response of a hierarchy structure starting at root.
+value:
+  _type: Collection
+  total: 37
+  count: 37
+  _embedded:
+    elements:
+      - _type: HierarchyItem
+        id: 1337
+        label: null
+        short: null
```

Comment:
This makes me think again if we should copy data to the root item, instead of keeping it only in the custom field.

I imagine that in most cases whoever is consuming the API will also have to fetch the custom field anyways, so this is probably a non issue.

Commenting just not to leave unsaid.